br/pkg/lightning/backend/external/util.go

```diff
@@ -292,3 +294,13 @@ func BytesMax(a, b []byte) []byte {
 	}
 	return b
 }
+
+func getSpeed(n uint64, dur float64, isBytes bool) string {
+	if dur == 0 {
+		return "-"
+	}
+	if isBytes {
+		return units.BytesSize(float64(n) / dur)
+	}
+	return units.HumanSize(float64(n) / dur)
```

Comment:
it will output B(Bytes) as unit when `isBytes` is false. I see the usage is to display kv/s, it should not have "B"